src/app/space/create/deployments/apps/deployment-card.component.ts

```diff
@@ -43,10 +44,12 @@ export class DeploymentCardComponent implements OnDestroy, OnInit {
   @Input() spaceId: string;
   @Input() applicationId: string;
   @Input() environment: Environment;
+  @Input() modal: DeleteDeploymentModal;
 
   active: boolean = false;
   detailsActive: boolean = false;
   collapsed: boolean = true;
+  deletingDeployment: boolean = false;
```

Comment:
I don't think the "deployment" needs to be part of this property's name - it already belongs to a component representing a deployment.